index.d.ts

```diff
@@ -28,21 +32,32 @@ export interface WebViewProps extends React.HTMLAttributes<Electron.WebViewEleme
   onDidGetResponseDetails?: EventListener
   onDidGetRedirectRequest?: EventListener
   onDomReady?: EventListener
-  onPageTitleSet?: EventListener
+  onPageTitleUpdated?: EventListener
```

Comment:
Do you know when this was changed? I'm thinking we leave them both in for compatibility, but with a comment to remove it later.